src/app/components/ThemeProvider/themes/polska.ts

```diff
@@ -0,0 +1,34 @@
+import { GHOST, WHITE, POSTBOX, POSTBOX_30 } from '../palette';
+import latinWithDiacriticsScript from '../fontScripts/latinWithDiacritics';
+import {
+  REITH_SANS_BOLD,
+  REITH_SANS_REGULAR,
+  REITH_SERIF_MEDIUM,
+  REITH_SERIF_LIGHT,
+} from '../fontFaces';
+import reithFontVariants from '../fontVariants/reith';
+import withThemeProvider from '../withThemeProvider';
+import brandSVG from '../chameleonLogos/portuguese';
```

Comment:
Worth creating `chameleonLogos/polska` and populating with the Portuguese stuff for now?

Reply:
Good shout, I can do that now.